### ISSUE TITLE
Use d.o api instead of parsing git log

### ISSUE THREAD

**Issue by lauriii** [lauriii/drupalcores#63]
Since Drupal.org has started collecting the contribution information for nodes, we could use that data through json api.

**Comment by ravindrasingh22** [comment]
Yes, this will help to get latest records without cron. second thing if the person got commit credit from reviewing the issue which is not implemented in the current system. it checks only patch has been contributed by people.

**Comment by lewisnyman** [comment]
The current code only parses the git log for credit, so everything that is decided to be credited by the commiter will get captured and should match the credit settings on each issue.

**Comment by ravindrasingh22** [comment]
Nopes, there are still something missing or error. for example in my git log there are 14 issues items are listed and when I check it drupalcores.com it shows only 13. http://cgit.drupalcode.org/drupal/log/?h=HEAD&qt=grep&q=RavindraSingh http://drupalcores.com/index.html#ravindrasingh
